packages/arb-token-bridge-ui/src/util/deposits/fetchDeposits.ts

```diff
@@ -74,60 +67,47 @@ export const fetchDeposits = async ({
     searchString
   })
 
-  const ethDepositsFromSubgraph: Transaction[] = depositsFromSubgraph.map(
-    (tx: FetchDepositsFromSubgraphResult) => {
-      const isEthDeposit = tx.type === 'EthDeposit'
-
-      const assetDetails = {
-        asset: 'ETH',
-        assetName: 'ETH',
-        assetType: AssetType.ETH,
-        tokenAddress: ''
-      }
-
-      if (!isEthDeposit) {
-        // update some values for token deposit
-        const symbol = tx.l1Token?.symbol || ''
-
-        assetDetails.asset = symbol
-        assetDetails.assetName = symbol
-        assetDetails.assetType = AssetType.ERC20
-        assetDetails.tokenAddress = tx?.l1Token?.id || ''
-      }
-
-      return {
-        type: 'deposit-l1',
-        status: 'pending',
-        value: utils.formatUnits(
-          (isEthDeposit ? tx.ethValue : tx.tokenAmount) || 0,
-          isEthDeposit ? 18 : tx?.l1Token?.decimals || 18
-        ),
-        txID: tx.transactionHash,
-        tokenAddress: assetDetails.tokenAddress,
-        sender: tx.sender,
-        destination: tx.receiver,
+  return depositsFromSubgraph.map(tx => {
+    const isEth = tx.type === 'EthDeposit'
 
-        asset: assetDetails.asset,
-        assetName: assetDetails.assetName,
-        assetType: assetDetails.assetType,
+    const assetDetails = {
+      asset: 'ETH',
+      assetName: 'ETH',
+      assetType: AssetType.ETH,
+      tokenAddress: ''
+    }
 
-        l1NetworkID: String(l1ChainId),
-        l2NetworkID: String(l2ChainId),
-        blockNumber: Number(tx.blockCreatedAt),
-        timestampCreated: tx.timestamp,
-        isClassic: tx.isClassic,
+    if (!isEth) {
+      // update some values for token deposit
+      const symbol = tx.l1Token?.symbol || ''
 
-        chainId: l2ChainId,
-        parentChainId: l1ChainId
-      }
+      assetDetails.asset = symbol
+      assetDetails.assetName = symbol
+      assetDetails.assetType = AssetType.ERC20
+      assetDetails.tokenAddress = tx?.l1Token?.id || ''
     }
-  )
-
-  const finalTransactions: Transaction[] = await Promise.all(
-    ethDepositsFromSubgraph.map(depositTx =>
-      updateAdditionalDepositData(depositTx, l1Provider, l2Provider)
-    )
-  )
 
-  return finalTransactions
+    return {
+      type: 'deposit-l1',
+      direction: 'deposit',
+      source: 'subgraph',
+      l1NetworkID: String(l1ChainId),
+      l2NetworkID: String(l2ChainId),
+      // TODO: may need to format units here
+      value: utils.formatUnits(
+        (isEth ? tx.ethValue : tx.tokenAmount) || 0,
+        isEth ? 18 : tx?.l1Token?.decimals || 18
```

Comment:
i think we want to treat the ones without decimals as 0?

Reply:
done